### PR TITLE
[SMALLFIX] Add warnings to validateEnv command

### PR DESCRIPTION
**Pull request by bf8086**
Some validateEnv tasks report problems that are not fatal. This change adds a new return type for validation tasks to indicate task result to be one of the 4 options: ok, warning, failed and skipped.

Bonus: the task results now have colors!

### CODE REVIEW

**Comment by bf8086** [comment]
@apc999 @yupeng9 PTAL. Thanks!

File: core/server/common/src/main/java/alluxio/cli/ValidateEnv.java [hunk 1]
```diff
@@ -244,25 +245,45 @@ private static boolean validateLocal(String target, String name, CommandLine cmd
...
-      System.err.format("Validation failed. Total failures: %d.%n", failureCount);
-      return false;
+    if (results.containsKey(ValidationTask.TaskResult.FAILED)) {
+      System.err.format("%d failures ", results.get(ValidationTask.TaskResult.FAILED));
+    }
+    if (results.containsKey(ValidationTask.TaskResult.WARNING)) {
+      System.err.format("%d warnings ", results.get(ValidationTask.TaskResult.WARNING));
     }
+    if (results.containsKey(ValidationTask.TaskResult.SKIPPED)) {
+      System.err.format("%d skipped ", results.get(ValidationTask.TaskResult.WARNING));
```
Comment (yupeng9) [inline core/server/common/src/main/java/alluxio/cli/ValidateEnv.java]: `results.get(ValidationTask.TaskResult. SKIPPED)`?

Comment (bf8086) [reply core/server/common/src/main/java/alluxio/cli/ValidateEnv.java]: Fixed.

**Comment by yupeng9** [comment]
Thanks for the improvement!

**Pull request merged**